datasets/cmu_hinglish_dog/cmu_hinglish_dog.py

```diff
@@ -180,10 +180,28 @@ def _generate_examples(self, hi_en_dir, en_dir):
                     x["text"] = re.sub("\t|\n", " ", x["text"])
                     y["text"] = re.sub("\t|\n", " ", y["text"])
                     line = {
-                        "uid": x["uid"],
-                        "docIdx": x["docIdx"],
-                        "hi_en": y["text"],
-                        "en": x["text"],
+                        "date": hi_en["date"],
+                        "history": {
+                            "uid": x["uid"],
+                            "docIdx": [x["docIdx"]],
+                            "utcTimestamp": x["utcTimestamp"],
+                            "text": x["text"] + "_" + y["text"],
```

Comment:
Since the type of input and output data types have to be the same, I had to output both Hinglish and English texts in a single text field (as input has only 1 text field).
Not to sure about this way of returning the texts seperated by a _. Please let me know if I am missing something or if there is an alternative :)